src/main/kotlin/app/Configurator.kt

```diff
@@ -164,16 +177,13 @@ object Configurator {
             return
         }
 
-        val mapper = ObjectMapper(YAMLFactory()) // Enable YAML parsing.
-        mapper.registerModule(KotlinModule()) // Enable Kotlin support.
-
         // Сonfig initialization in case an exception is thrown.
         var loadConfig = Config()
 
         try {
             loadConfig = Files.newBufferedReader(Paths.get(userDir,
                     CONFIG_FILE_NAME)).use {
-                mapper.readValue(it, Config::class.java)
+                createMapper().readValue(it, Config::class.java)
```

Comment:
Is this mapper creation thing expensive? If it is, you could create it once and store it in a field.